### DISCUSSION TITLE
[RFC] Switch from `django-notifications` to `django-activity-stream`

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#1434] [Ideas]
We currently use the [`django-notifications` package](https://github.com/django-notifications/django-notifications) in this project. Recently, there are some deprecation warnings in `django-notifications` that have led for us to investigate the package in issue #1426.

The `django-notifications` documentation indicates that it may be a fork of the [`django-activity-stream` package](https://github.com/justquick/django-activity-stream).

Since `django-notifications` and `django-activity-stream` are so similar, let's compare the two packages (code quality, development activity, documentation, etc.) and discuss whether to stay with `django-notifications` or switch to `django-activity-stream` as well as consider the possible switching cost.

**Comment by DamyanBG** [comment]
I will test django-activity-stream and I will write my feedback.

**Comment by DamyanBG** [comment]
From the first view I can see that the django-notifications are used in two python files - accounts/api.py and threads/api.py. I think it will be not so much work to switch to django-activity-stream. I can see that django-activity-stream have better support, but I do not think it have big difference. My personal opinion is that there is not big value to switch to django-activity-stream.
